src/components/point-tracker-summary/point-tracker-summary.js

```diff
@@ -41,22 +41,24 @@ class PointTrackerSummary extends React.Component {
               <h4>{pointTracker.title}</h4>
               <br />
               <span className="title">
-                Playing Time Earned:
+                Playing Time Earned
               </span>
               <span> {pointTracker.earnedPlayingTime}</span>
               <br />
               <span className="title">
-                Mentor Granted Playing Time:
+                Mentor Granted Playing Time
               </span>
               <span>{pointTracker.mentorGrantedPlayingTime}</span>
               <p>{pointTracker.synopsisComments.mentorGrantedPlayingTimeComments}</p>
               <br />
-              <span className="title">Student Action Items:</span>
+              <span className="title">Student Action Items</span>
               <p>{pointTracker.synopsisComments.studentActionItems}</p>
+              <br />
+              <span className="title">Full Synposis Report on Google Drive</span>
+              <a href={pointTracker.synopsisLink} target="_blank" rel="noopener noreferrer">{pointTracker.synopsisLink}</a>
```

Comment:
The anchor text doesn't have to be the URL itself. The URL is pretty gnarly.  I'd just use plain text. "Full Synopsis Report on Google Drive" or something like that.